mmdeploy/apis/visualize.py

```diff
@@ -71,21 +71,29 @@ def visualize_model(model_cfg: Union[str, mmengine.Config],
     with torch.no_grad():
         result = model.test_step(model_inputs)[0]
 
+    visualize = True
     try:
         # check headless
         import tkinter
         tkinter.Tk()
+    except Exception as e:
+        from mmdeploy.utils import get_root_logger
+        logger = get_root_logger()
+        logger.warn(
```

Comment:
`logger.warn` is deprecated, better use `logger.warning`

Reply:
done.